exercises/build.gradle

```diff
@@ -9,14 +9,14 @@ subprojects { project ->
       java.srcDirs = ["src/example/java"]
     }
     project["compileJava"].doFirst { compileTask ->
-      println "  (source = " + compileTask.source.asPath + ")"
+      println "  (source = ${compileTask.source.asPath})"
     }
 
     starterSource {
       java.srcDirs = ["src/main/java"]
     }
     project["compileStarterSourceJava"].doFirst { compileTask ->
-      println "  (source = " + compileTask.source.asPath + ")"
+      println "  (source = ${compileTask.source.asPath})"
```

Comment:
For something short like `$compileTask` I normally omit the braces, but with a longer composite invocation such as this one, I prefer them for clarity of scoping, as you suggested :)